### DISCUSSION TITLE
Add aarch64 package to Cloudsmith for Debian

### DISCUSSION THREAD

**Discussion by tobsen111** [Kong/kong#12050] [Help]
Hi,

would it be possible to also add the aarch64 binary for Debian to Cloudsmith.

Thanks in advance.

**Comment by chronolaw** [comment]
@fffonion , do you have any idea to share with us?

**Reply by fffonion** [reply]
That will be a product decision to made. cc @teb510

**Comment by tobsen111** [comment]
Hi,
is there any update if it would be possible to upload the Debian arm package also to Cloudsmith? At the moment there are only packages for ubuntu tagged with arm64. @teb510 @fffonion @chronolaw 

Thanks ;)
